src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,3 @@
+$header-nav-link-hover-color: #00acdc;
+$card-text-hover-color: #34568b;
```

Comment:
the names are too specific, you want them more generic in your code.
like `primary-green` or sth like that. Because you can't reuse `$header-nav-link-hover-color` in some place, which is not in the header. But usually on your project you don't create a custom color for everything, you reuse existing ones